client-v2/src/components/Clipboard.tsx

```diff
@@ -211,24 +218,27 @@ class Clipboard extends React.Component<ClipboardProps> {
                           onDrop={this.handleInsert}
                           displayType="polaroid"
                         >
-                          {(supporting, sProps) => (
-                            <CollectionItem
-                              uuid={supporting.uuid}
-                              frontId={clipboardId}
-                              parentId={articleFragment.uuid}
-                              getNodeProps={() => sProps}
-                              size="small"
-                              displayType="polaroid"
-                              onSelect={id =>
-                                this.props.selectArticleFragment(id, true)
-                              }
-                              onDelete={() =>
-                                this.props.removeSupportingCollectionItem(
-                                  articleFragment.uuid,
-                                  supporting.uuid
-                                )
-                              }
-                            />
+                          {(supporting, sProps, i, arr) => (
```

Comment:
Could `arr` be named more clearly? I guess it refers to all elements in the list?

Reply:
(obviously not a blocker, it caught my eye just as I'd approved!)

Reply:
I missed this, I just treated it like the `arr` arg in a `Array#map` so named it like it's named in MDN. But you're right it probably should be clearer - I should probably do that from now on 😞